corebc-core/src/utils/anvil.rs

```diff
@@ -45,7 +45,7 @@ impl AnvilInstance {
 
     /// Returns the network of the anvil instance
     pub fn network_id(&self) -> u64 {
-        self.network_id.unwrap_or_else(|| Network::AnvilHardhat.into())
+        self.network_id.unwrap_or_else(|| Network::Devin.into())
     }
```

Comment:
This isn't really correct. AnvilHardhat type was a local instance of a blockchain. Devin is a testnet. But for now since we don't have any local blockchain emulator, it is probably fine. But we should probably add a comment explaining this.

Something like: // Should be a local node instead of Devin